docs/manual/common/concepts/InternalAndExternalCommunication.md

```diff
@@ -1,26 +1,40 @@
 
-* intra-service communication --- between multiple instances of the same service.
+The following topics discuss these communication paths in more detail:
 
-* inter-service communication --- between different services in the same system.
+* [Communication within a microservices system](#Communication-within-a-microservices-system)
+* [Communication with parties outside of a microservices system](#Communication-with-parties-outside-of-a-microservices-system)
 
-* extra-service communication --- between legacy systems and services outside of your system. 
+## Communication within a microservices system 
 
-Within your Lagom system, inter-service and intra-service communication should be asynchronous. Consider a cluster of multiple instances of the same microservice. Each instance needs to keep up to date with work being processed by other instances without blocking. The same principle applies to communication between different services. The Publish-Subscribe model makes it easy to accomplish this. Lagom supports Publish-Subscribe with two levels of guarantees:
+While similar in principle, inter- and intra-service communication have very different needs, and Lagom offers multiple implementation options. Inter-service communication must use loosely-coupled protocols and message formats to maintain isolation and autonomy. Coordinating change between different services can be difficult and costly. You can achieve this in your system by taking advantage of the following:
 
-* [[Publish-Subscribe|PubSub]] --- for at-most-once semantics. This real-time model supports services that do not care about past events, such as messaging, gaming, auction, sensors, or stock tickers. Limitations include:
+* [[Service calls|ServiceDescriptors]], either synchronous or asynchronous (streaming), allow services to communicate with each other using published APIs and standard protocols (HTTP and WebSockets).
+
+* Publishing messages to a [[broker|MessageBroker]], such as Apache Kafka, decouples communication even further. Lagom's [[Message Broker API|MessageBrokerAPI]] provides at-least-once semantics. If a new instance starts publishing information, its messages are added to events previously emitted. If a new instance subscribes to a topic, they will receive all events, past, present, and future (as long as they are subscribed).
+
+Nodes of a single service (collectively called a cluster) require less decoupling. They share the same code and are managed together, as a set, by a single team or individual. For this reason, intra-service communication can take advantage of mechanisms that have less overhead and better performance. For example:
+
+* Many Lagom components use Akka remoting internally, and you can use it directly in your services.
```

Comment:
Should we link Akka remoting to a reference such as http://doc.akka.io/docs/akka/2.4/general/remoting.html?